### DISCUSSION TITLE
PWA Build Looking for Files in Wrong Directory

### DISCUSSION THREAD

**Discussion by CynthiaBlue** [quasarframework/quasar#12192] [CLI - PWA mode] [answered]
When I build my Quasar app and upload it to my server, I am getting some file not found (404) errors.  I'm not sure why, my other application I've built and run on my server works... hmm, (just had a thought) that one is loaded to the main directory of my server, and this one I want in a subdirectory.  Maybe that is the difference.

Anyway, this is a screenshot of my console showing the Network tab and the errors.  The URL I am putting the built code into is:
[https://pawtap.net/pawtapNew/](url), and as you can see, it's looking for some of the files in just pawtap.net.  So it should be looking in https://pawtap.net/pawtapNew/js/vendor.e47c046d.js

This is a live website so if you have the inkling feel free to go look. :)

Any advice about what I've done to mess this up? I really don't know.  I have primarily compiled this for Electron. But I want to be able to run it both with Electron and as a PWA app on my server.

<img width="764" alt="Screen Shot 2022-01-25 at 11 53 21 AM" src="https://user-images.githubusercontent.com/34698014/151041015-becb5900-f009-48b9-a188-c679567ba6a5.png">

Thanks!

**Comment by MilosPaunovic** [comment, marked as answer]
Check out the `publicPath` property:
[https://quasar.dev/quasar-cli/quasar-conf-js#property-build](https://quasar.dev/quasar-cli/quasar-conf-js#property-build)

**Reply by CynthiaBlue** [reply]
THANK YOU! That worked. :)
